portable-atomic-util/tests/arc.rs

```diff
@@ -652,11 +658,19 @@ mod alloc_tests {
     fn test_downcast() {
         use std::any::Any;
 
-        // TODO: CoerceUnsized is needed to cast to Arc<dyn ..>
-        // let r1: Arc<dyn Any + Send + Sync> = Arc::new(i32::MAX);
-        // let r2: Arc<dyn Any + Send + Sync> = Arc::new("abc");
+        #[cfg(portable_atomic_unstable_coerce_unsized)]
+        let r1: Arc<dyn Any + Send + Sync> = Arc::new(i32::MAX);
+        // TODO: This is a workaround in case CoerceUnsized is not available - remove once this is no longer needed
+        #[cfg(not(portable_atomic_unstable_coerce_unsized))]
         let r1: Arc<dyn Any + Send + Sync> =
             Arc::from(Box::new(i32::MAX) as Box<dyn Any + Send + Sync>);
+
+        // XXX INJECT DASH SIGN - SEE IF CI FAILS AS EXPECTED WITH THIS INCORRECT VALUE
+        // XXX TODO NEED TO REMOVE DASH SIGN FOR CI TEST TO SUCCEED
+        #[cfg(portable_atomic_unstable_coerce_unsized)]
+        let r2: Arc<dyn Any + Send + Sync> = Arc::new("-abc");
+        // TODO: This is a workaround in case CoerceUnsized is not available - remove once this is no longer needed
+        #[cfg(not(portable_atomic_unstable_coerce_unsized))]
```

Comment:
NEED TO FIX THIS TEST CASE once CI output shows THIS KNOWN FAILURE with `#[cfg(portable_atomic_unstable_coerce_unsized)]` option:

```suggestion
        #[cfg(portable_atomic_unstable_coerce_unsized)]
        let r1: Arc<dyn Any + Send + Sync> = Arc::new(i32::MAX);
        // TODO: This is a workaround in case CoerceUnsized is not available - remove once this is no longer needed
        #[cfg(not(portable_atomic_unstable_coerce_unsized))]
        let r1: Arc<dyn Any + Send + Sync> =
            Arc::from(Box::new(i32::MAX) as Box<dyn Any + Send + Sync>);

        #[cfg(portable_atomic_unstable_coerce_unsized)]
        let r2: Arc<dyn Any + Send + Sync> = Arc::new("abc");
        // TODO: This is a workaround in case CoerceUnsized is not available - remove once this is no longer needed
        #[cfg(not(portable_atomic_unstable_coerce_unsized))]
```